<?xml version="1.0"?>
<launch>
    <!--argument-->
<!--    <arg name="low_lines" default="4" />-->
    <arg name="z_threshold" default="-2.5" />
    <arg name="ransac_distance_threshold" default="0.04" />
    <arg name="map_tf" default="velo_middle" />
<!--    <arg name="lowest_segment_point_num" default="7" />-->


    <node pkg="ground_detect" type="local_env_provider" name="local_env_provider_node" output="screen">
<!--        <remap from="/lidar/vlp32_middle/PointCloud2" to="/lidar/vlp32_middle/PointCloud2_compensated" />-->
<!--        <param name="low_lines" value="$(arg low_lines)" />-->
        <param name="z_threshold" value="$(arg z_threshold)" />
<!--        <param name="distance_threshold" value="$(arg distance_threshold)" />-->
    </node>

    <node pkg="ground_detect" type="ransac_get_plane" name="ransac_get_plane_node" output="screen">
        <param name="distance_threshold" value="$(arg ransac_distance_threshold)" />
<!--&lt;!&ndash;        <param name="z_threshold" value="$(arg z_threshold)" />&ndash;&gt;-->
<!--        <param name="distance_threshold" value="$(arg distance_threshold)" />-->
<!--        <param name="lowest_segment_point_num" value="$(arg lowest_segment_point_num)" />-->
    </node>

    <!--rviz-->
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find ground_detect)/rviz/ransac_plane.rviz -f $(arg map_tf)" />
</launch>
